built_value/lib/serializer.dart

```diff
@@ -67,6 +68,7 @@ abstract class Serializers {
           ..add(new Int64Serializer())
           ..add(new JsonObjectSerializer())
           ..add(new NumSerializer())
+          ..add(RegExpSerializer())
```

Comment:
`built_value` is still compatible with Dart 1, for a few legacy uses ... please keep the `new` :)